pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.black]
+line-length = 120
+skip-string-normalization = true
+target-version = ['py39']
+include = "src/vorta/.*.py$"
```

Comment:
Any reason tests aren't formatted (while isort runs on them too)?

Reply:
No, just an oversight on my part. We can sneak that change in with another PR.

Reply:
I can put it into #1394.